addons/account_peppol/wizard/account_move_send.py

```diff
@@ -118,12 +118,12 @@ def _call_web_service(self, prepared_data_list):
                 # should never happen but in case it does, we need to handle it
                 move.peppol_move_state = 'error'
                 move._message_log(body=_('The partner is missing Peppol EAS and/or Endpoint identifier.'))
-                continue
+                raise UserError(_('The partner is missing Peppol EAS and/or Endpoint identifier.'))
```

Comment:
Btw, I think the message is already logged into the invoice in case of error:
https://github.com/odoo/odoo/blob/master/addons/account/wizard/account_move_send.py#L382